.github/workflows/build-and-push-image-development.yml

```diff
@@ -40,4 +40,21 @@ jobs:
           context: .
           file: ./Dockerfile
           push: true
-          tags: ${{ steps.prepare-tags.outputs.tags }}
\ No newline at end of file
+          tags: ${{ steps.prepare-tags.outputs.tags }}
+
+      - name: Azure login with ACA credentials
```

Comment:
When does git get tagged?

Reply:
This repository isn't currently using tags, so these workflows will not automate the creation of tags